back-end/aws-cloud9-api/src/main/java/com/serverless/QuestionHandlers/CreateMembershipHandler.java

```diff
@@ -0,0 +1,62 @@
+package com.serverless.QuestionHandlers;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.Map;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.serverless.ApiGatewayResponse;
+import com.serverless.Handler;
+import com.serverless.Response;
+import com.serverless.dal.Form;
+import com.serverless.dal.FormDBTable;
+import com.serverless.dal.Question;
+import com.serverless.dal.QuestionDBTable;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.List;
+public class CreateMembershipHandler implements RequestHandler<Map<String, Object>, ApiGatewayResponse> {
+
+	private static final Logger LOG = LogManager.getLogger(Handler.class);
+
+	@Override
+	public ApiGatewayResponse handleRequest(Map<String, Object> input, Context context) {
+		try {
+			JsonNode body = new ObjectMapper().readTree((String) input.get("body"));
+			String questionid=(body.get("idquestion").asText());
+			String formid=(body.get("idform").asText());
+			QuestionDBTable question = new QuestionDBTable(); //form i table
+			boolean work=question.createMembership(questionid, formid);	// czy zadzialalo
+
+			
+		
+			return ApiGatewayResponse.builder()
+      				.setStatusCode(200)
+      				.setObjectBody(work)
+      				.setHeaders(Collections.singletonMap("X-Powered-By", "AWS Lambda & Serverless"))
+      				.build();
+			
+		} catch (IOException e) {
+			Response responseBody = new Response("lipa", input);
+			return ApiGatewayResponse.builder()
+					.setStatusCode(500)
+					.setObjectBody(responseBody)
+					.setHeaders(Collections.singletonMap("X-Powered-By", "AWS Lambda & serverless"))
+					.build();
+		}
+		
+		
+		
+		
+		
+	}
+}
```

Comment:
Potrzebne? Można to robić przy tworzeniu obiektu, albo ogólnie przy modyfikacji.

Reply:
tak, można przypisać pytanie do innego testu niż na początku